.github/workflows/ci-build.yaml

```diff
@@ -68,7 +68,20 @@ jobs:
         run: |
           make
 
+      - name: Build docker image
+        run: |
+          sudo git clone https://github.com/l3af-project/l3af-arch.git /root/l3af-arch
+          sudo cp /root/l3af-arch/dev_environment/cfg/l3afd.cfg ./build-docker
```

Comment:
Avoid cloning of l3af-arch repo, its better to have local copy of l3afd.cfg file here.